### DISCUSSION TITLE
How to access my files

### DISCUSSION THREAD

**Discussion by TheArKaID** [minio/minio#13375] [Q&A] [answered]
Hi, I'm totally new to Storage S3 (never even used AWS S3). So, I've installed MinIO (I didn't install it), and a bucket with public access policy, then how do I access that file? Like placing it on the frontend or calling it via the browser ? I'm quite confused about where to start. 

Thanks.

![image](https://user-images.githubusercontent.com/35050722/136312783-1d4da523-eaa4-439a-b06a-9b40fc170a8a.png)

**Comment by klauspost** [comment, marked as answer]
S3 is not just a file dump (you can use an HTTP server for that). To access it, use S3 API or S3 compatible applications: https://aws.amazon.com/s3/getting-started/
